test/test-util.js

```diff
@@ -121,8 +121,9 @@ function testStateTransitions(ns, initial, newState, finalState, incNoDelta, sta
 function prepareCluster(insert_fns) {
     return function(t, tc, n) {
         return [
-            dsl.assertDetectChecksumMethod(t, tc),
+            dsl.waitForPing(t, tc, false),
```

Comment:
Maybe we should describe why we are waiting for a ping, and why we don't consume it. Both are very subtile but important facts.